www/source/partials/docs/_dev-pkgs-exports.html.md.erb

```diff
@@ -117,7 +117,30 @@ The Kubernetes exporter is an additional command line subcommand to the standard
 
 4. More information and a demo video is available on the [announcement blog](https://kinvolk.io/blog/2017/12/introducing-the-habitat-kubernetes-exporter/)
     
+## Export to a Helm chart
```

Comment:
Just an idea for the future:

The wording here makes me think we should not call these exporters but rather generaters? :D 

As we only generate the files rather then export an image, binary or anything like that. Maybe we should add an option to generate the Helm package, that way it will be an exporter? WDYT?

Reply:
I think both terms are correct, "export" always implies generating something(s). The dir provided is readily consumable, if you passed the `--downnload-deps` option.

Having said that, yeah, that option makes sense and was already on the todo.